helpers/test-utils.tsx

```diff
@@ -40,3 +41,21 @@ export function createMatchMedia(width: number) {
     removeListener: jest.fn(),
   })
 }
+
+/**
+ * Mock Formik for rendering Formik components when you don't care about the formik connection.
+ */
+export const MockFormik = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <Formik
+      initialValues={{}}
+      onSubmit={
+        // This isn't meant to be used so it won't be tested
+        /* istanbul ignore next */
```

Comment:
It's possible to turn such suppression comments into one-liners:

```suggestion
        /* istanbul ignore next; this isn't meant to be used so it won't be tested */
```

Reply:
Ah nice